Dockerfile

```diff
@@ -45,10 +45,20 @@ RUN make build
 
 # Run image
 FROM quay.io/konveyor/move2kube:${VERSION}
+WORKDIR /move2kube-api
 ENV MOVE2KUBE_PLATFORM="${MOVE2KUBE_PLATFORM}:api-dockerfile"
+
 # Install move2kube-api
 COPY --from=build_base /go/bin/move2kube-api /bin/move2kube-api
+# Copy static files for Move2Kube UI
+COPY ./dist ./dist
+
+# for openshift
+RUN chmod -R 775 /move2kube-api
+RUN chown -R 12345:root /move2kube-api
+USER 12345
```

Comment:
Instead we can change it to be executable by any user

Reply:
done